src/commonMain/kotlin/fr/acinq/lightning/serialization/v1/ChannelState.kt

```diff
@@ -695,7 +695,8 @@ data class Normal(
         channelUpdate,
         remoteChannelUpdate,
         localShutdown,
-        remoteShutdown
+        remoteShutdown,
+        null
```

Comment:
Do you think we should introduce a new version to persist those?
I had to use `process` instead of `processEx` in some tests because the closing feerates are currently ignored when storing the state.

Reply:
Note to self: we should probably use #277 that introduces a new version to also include this new field (as it will probably be merged after this PR).